From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lemures Lemniscati <lemures.lemniscati@gmail.com>
Date: Tue, 20 Oct 2020 21:21:21 +0900
Subject: [PATCH] Make PYTHON_INSTDIR configurable


diff --git a/scripts/CMakeLists.txt b/scripts/CMakeLists.txt
index c01cacf6c..a0e845133 100644
--- a/scripts/CMakeLists.txt
+++ b/scripts/CMakeLists.txt
@@ -103,6 +103,7 @@ if (DO_PYTHON_BINDINGS)
             OUTPUT_NAME _openbabel
             PREFIX ""
             SUFFIX .so )
+      if( "${PYTHON_INSTDIR}" STREQUAL "" )
         execute_process(
           COMMAND
           ${PYTHON_EXECUTABLE} -c "from distutils import sysconfig; print(sysconfig.get_python_lib(1,0,prefix='${CMAKE_INSTALL_PREFIX}'))"
@@ -119,6 +120,7 @@ if (DO_PYTHON_BINDINGS)
           )
           set(PYTHON_INSTDIR "${PYTHON_INSTDIR}/dist-packages")
 	endif()
+      endif()
         if(NOT BINDINGS_ONLY)
             add_dependencies(bindings_python openbabel)
         endif()
