From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lemures Lemniscati <lemures.lemniscati@gmail.com>
Date: Tue, 20 Oct 2020 21:21:22 +0900
Subject: [PATCH] Make JAVA_INSTDIR configurable


diff --git a/scripts/CMakeLists.txt b/scripts/CMakeLists.txt
index a0e845133..5abfe53ce 100644
--- a/scripts/CMakeLists.txt
+++ b/scripts/CMakeLists.txt
@@ -235,11 +235,14 @@ include_directories(
       add_dependencies(bindings_java openbabel)
     endif()
 
+    if( "${JAVA_INSTDIR}" STREQUAL "" )
+      set( JAVA_INSTDIR "${LIB_INSTALL_DIR}" )
+    endif()
     install(TARGETS bindings_java
-            LIBRARY DESTINATION ${LIB_INSTALL_DIR}
+            LIBRARY DESTINATION "${JAVA_INSTDIR}"
             COMPONENT bindings_java)
     install(FILES ${openbabel_SOURCE_DIR}/scripts/java/openbabel.jar
-            DESTINATION ${LIB_INSTALL_DIR}
+            DESTINATION "${JAVA_INSTDIR}"
             COMPONENT bindings_java)
 
 endif (DO_JAVA_BINDINGS)
