From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lemures Lemniscati <lemures.lemniscati@gmail.com>
Date: Tue, 20 Oct 2020 21:21:23 +0900
Subject: [PATCH] Make R_INSTDIR configurable


diff --git a/scripts/CMakeLists.txt b/scripts/CMakeLists.txt
index 5abfe53ce..ddcf181ba 100644
--- a/scripts/CMakeLists.txt
+++ b/scripts/CMakeLists.txt
@@ -293,11 +293,14 @@ if (DO_R_BINDINGS)
 		 add_dependencies(bindings_R openbabel)
     endif()
 
+    if( "${R_INSTDIR}" STREQUAL "" )
+      set( R_INSTDIR "${LIB_INSTALL_DIR}" )
+    endif()
 	 install(TARGETS bindings_R
-            LIBRARY DESTINATION ${LIB_INSTALL_DIR}
+            LIBRARY DESTINATION "${R_INSTDIR}"
 				COMPONENT bindings_R)
 	 install(FILES ${openbabel_BINARY_DIR}/scripts/R/openbabel_R.R
-            DESTINATION ${LIB_INSTALL_DIR}
+            DESTINATION "${R_INSTDIR}"
 				COMPONENT bindings_R)
 
 endif (DO_R_BINDINGS)
