From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lemures Lemniscati <lemures.lemniscati@gmail.com>
Date: Tue, 20 Oct 2020 21:21:25 +0900
Subject: [PATCH] Make RUBY_INSTDIR configurable


diff --git a/scripts/CMakeLists.txt b/scripts/CMakeLists.txt
index 36b19801c..3c9444942 100644
--- a/scripts/CMakeLists.txt
+++ b/scripts/CMakeLists.txt
@@ -441,8 +441,11 @@ if (DO_RUBY_BINDINGS)
       add_dependencies(bindings_ruby openbabel)
     endif()
 
+    if( "${RUBY_INSTDIR}" STREQUAL "" )
+      set( RUBY_INSTDIR "${LIB_INSTALL_DIR}" )
+    endif()
     install(TARGETS bindings_ruby
-            LIBRARY DESTINATION ${LIB_INSTALL_DIR}
+            LIBRARY DESTINATION ${RUBY_INSTDIR}
             COMPONENT bindings_ruby)
 
 endif (DO_RUBY_BINDINGS)
