From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lemures Lemniscati <lemures.lemniscati@gmail.com>
Date: Tue, 20 Oct 2020 21:21:26 +0900
Subject: [PATCH] Make CSHARP_INSTDIR configurable


diff --git a/scripts/CMakeLists.txt b/scripts/CMakeLists.txt
index 3c9444942..a1a9414ce 100644
--- a/scripts/CMakeLists.txt
+++ b/scripts/CMakeLists.txt
@@ -497,8 +497,11 @@ if (DO_CSHARP_BINDINGS)
     target_link_libraries(bindings_csharp ${BABEL_LIBRARY})
     set_target_properties(bindings_csharp PROPERTIES
         OUTPUT_NAME openbabel_csharp )
-    install(TARGETS bindings_csharp LIBRARY DESTINATION ${LIB_INSTALL_DIR})
-    install(FILES ${openbabel_SOURCE_DIR}/scripts/csharp/OBDotNet.dll DESTINATION ${LIB_INSTALL_DIR})
+    if( "${CSHARP_INSTDIR}" STREQUAL "" )
+      set( CSHARP_INSTDIR "${LIB_INSTALL_DIR}" )
+    endif()
+    install(TARGETS bindings_csharp LIBRARY DESTINATION "${CSHARP_INSTDIR}")
+    install(FILES ${openbabel_SOURCE_DIR}/scripts/csharp/OBDotNet.dll DESTINATION "${CSHARP_INSTDIR}")
 
 endif (DO_CSHARP_BINDINGS)
 
