From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lemures Lemniscati <lemures.lemniscati@gmail.com>
Date: Tue, 20 Oct 2020 21:21:27 +0900
Subject: [PATCH] Make PHP_INSTDIR configurable


diff --git a/scripts/CMakeLists.txt b/scripts/CMakeLists.txt
index a1a9414ce..d62c754f3 100644
--- a/scripts/CMakeLists.txt
+++ b/scripts/CMakeLists.txt
@@ -582,11 +582,15 @@ if (DO_PHP_BINDINGS)
     install(TARGETS bindings_php
             LIBRARY DESTINATION ${PHP_EXTENSION_DIR}
 	    COMPONENT bindings_php)
+
+    if( "${PHP_INSTDIR}" STREQUAL "" )
+      set( PHP_INSTDIR "${LIB_INSTALL_DIR}" )
+    endif()
     install(FILES ${openbabel_SOURCE_DIR}/scripts/php/openbabel.php
-            DESTINATION ${LIB_INSTALL_DIR}
+            DESTINATION "${PHP_INSTDIR}"
 	    COMPONENT bindings_php)
-	install(FILES ${openbabel_SOURCE_DIR}/scripts/php/baphpel.php
-	        DESTINATION ${LIB_INSTALL_DIR}
+    install(FILES ${openbabel_SOURCE_DIR}/scripts/php/baphpel.php
+            DESTINATION "${PHP_INSTDIR}"
 	    COMPONENT bindings_php)
 
 endif (DO_PHP_BINDINGS)
