--- origsrc/openbabel-f03cea648ffb0ca823ad01b708e26820f09ed002/scripts/CMakeLists.txt
+++ src/openbabel-f03cea648ffb0ca823ad01b708e26820f09ed002/scripts/CMakeLists.txt
@@ -103,11 +103,19 @@
     endif()
 
     if(NOT WIN32)
+      if(CYGWIN)
+        set_target_properties(bindings_python PROPERTIES
+            OUTPUT_NAME _openbabel
+            PREFIX ""
+            LIBRARY_OUTPUT_DIRECTORY "${openbabel_SOURCE_DIR}/scripts/python/openbabel/"
+            )
+      else()
         set_target_properties(bindings_python PROPERTIES
             OUTPUT_NAME _openbabel
             PREFIX ""
             LIBRARY_OUTPUT_DIRECTORY "${openbabel_SOURCE_DIR}/scripts/python/openbabel/"
             SUFFIX .so )
+      endif()
       if( "${PYTHON_INSTDIR}" STREQUAL "" )
         execute_process(
           COMMAND
