--- origsrc/openbabel-2.3.1/src/GUI/OBGUI.rc	2009-08-07 11:53:14.000000000 -0500
+++ src/openbabel-2.3.1/src/GUI/OBGUI.rc	2012-01-01 20:47:15.399596800 -0600
@@ -7,5 +7,7 @@ aaaaaaaa ICON "babel2.ico"
 
 // this icon is used with wxFrame::SetIcon()
 sample ICON "babel2.ico"
+#ifdef __WXMSW__
 #include "wx/msw/wx.rc"
+#endif
 
--- origsrc/openbabel-2.3.1/test/CMakeLists.txt	2011-01-11 08:53:12.000000000 -0600
+++ src/openbabel-2.3.1/test/CMakeLists.txt	2012-01-01 21:08:46.925911200 -0600
@@ -170,7 +170,7 @@ install(TARGETS roundtrip
 # The following tests should work okay in MinGW, it's just that the calling
 # script needs some work. On Cygwin, there's some weird DLL problems when
 # calling from Python.
-if(NOT MINGW AND NOT CYGWIN)
+if(NOT MINGW)
   include(UsePythonTest)
   if(PYTHON_EXECUTABLE)
     set(pytests
@@ -186,7 +186,7 @@ if(NOT MINGW AND NOT CYGWIN)
       )
     endforeach(pytest ${pytests})
   endif(PYTHON_EXECUTABLE)
-endif(NOT MINGW AND NOT CYGWIN)
+endif(NOT MINGW)
 
 ###############################
 # Tests using Python bindings #
