--- origsrc/openbabel-2.3.2/scripts/CMakeLists.txt	2015-05-20 18:35:17.318060800 -0500
+++ src/openbabel-2.3.2/scripts/CMakeLists.txt	2015-05-20 18:37:07.243019500 -0500
@@ -303,7 +303,7 @@ if (DO_RUBY_BINDINGS)
         # no regex metacharacters except possibly '.':
         string(REGEX REPLACE ^${RUBY_POSSIBLE_LIB_DIR}/ruby ${RUBY_PREFIX} ruby_install_dir "${RUBY_SITEARCH_DIR}")
       else (RUBY_PREFIX)
-        execute_process(COMMAND ${RUBY_EXECUTABLE} -r rbconfig -e "print Config::CONFIG['exec_prefix']"
+        execute_process(COMMAND ${RUBY_EXECUTABLE} -r rbconfig -e "print RbConfig::CONFIG['exec_prefix']"
           OUTPUT_VARIABLE ruby_exec_prefix)
         # It is reasonably safe to assume that ruby_exec_prefix contains no
         # regex metacharacters except possibly '.':
--- origsrc/openbabel-2.3.2/scripts/ruby/extconf.rb	2015-05-20 18:35:17.156540300 -0500
+++ src/openbabel-2.3.2/scripts/ruby/extconf.rb	2015-05-20 18:37:24.620226100 -0500
@@ -3,7 +3,7 @@
 # Compensate for the fact that Ruby will try to build universal
 # binaries on OS X by default
 require 'rbconfig'
-if Config::CONFIG["arch"] =~ /universal-darwin/
+if RbConfig::CONFIG["arch"] =~ /universal-darwin/
   ENV['ARCHFLAGS'] = case `uname -smr`.chomp
     when "i386" then '-arch i386'
     when "ppc"  then '-arch ppc'
